README.md

```diff
@@ -1,15 +1,43 @@
-# PANCHANGE - analysis scripts
+# PANCHANGE - factors influencing perceived and objective change in symptoms of anxiety, depression and PTSD during 2020 pandemic in the UK
+
+## Summary
+
 Code repository for Young et al 2020 manuscript "Anxiety, depression and trauma symptom change during the COVID-19 pandemic: Retrospective versus objective assessment"
 
+The project uses data from the [Repeated Assessment of Mental health during a Pandemic (RAMP)](https://rampstudy.co.uk/), COVID-19 Psychiatry and Neurological Genetics (COPING), [Genetic links to Anxiety and Depression](https://gladstudy.org.uk/) (GLAD) and [Eating Disorder Genetics Initiative](https://edgiuk.org/) (EDGI) studies.
+
+***Aims***
+1) Assess current symptoms among individuals with and without a prior diagnosis of an anxious or depressive disorder.  
+2) Assess objective and retrospective symptom change in the GLAD sample.   
+3) If 2. considered sufficiently valid, report on whole RAMP and COPING sample current v retrospective change
+
+***Sensitivity checks / stratifications***
+* perceived change in symptoms
+* Pre-existing mental health diagnosis
+* Chronicity of pre-existing diagnosis 
+* Severity of objective baseline symptoms (GLAD & EDGI)
+* Gender
+* Age
+* Ethnicity (if sufficiently powered to do so)
+* Keyworker / vulnerable individual status
+* COVID-19 status (did they become ill, are they at the time or response)
```

Comment:
@klpurves: The COVID algorithm includes sex and age, so we will need to discuss how to do this for RAMP and should probably run sensitivity analyses for the COPING sample to see how much the RAMP-adapted algorithm differs from the original (since we can run algorithms with/without sex + exact age in COPING). 

@topherhuebel has added this to the meeting agenda for tomorrow to discuss further, and I'll add it as an issue as well.